modules/samples/doormanager/component/controller/src/main/java/org.homeautomation/doormanager/controller/api/DoorManagerControllerService.java

```diff
@@ -76,6 +77,8 @@
 
     private PrivilegedCarbonContext ctx;
 
+    @Context  //injected response proxy supporting multiple thread
+
```

Comment:
Fixed in commit https://github.com/wso2/product-iots/commit/1f06c08f0d87e3161e7a135e9d4496e757460247